src/vcpkg/base/json.cpp

```diff
@@ -538,7 +538,7 @@ namespace vcpkg::Json
                 // cur == '\\'
                 if (at_eof())
                 {
-                    add_error("Unexpected EOF after escape character");
+                    add_error(msg::format(msgUnexpectedEOF));
```

Comment:
```suggestion
                    add_error(msg::format(msgUnexpectedEOFAfterEscape));
```

(and update other places to match of course)